redfish.conf

```diff
@@ -3,6 +3,7 @@ schema_dir = schemas
 registry_dir = registry
 indent_json = True
 xml_prettify = True
+host = 0.0.0.0
```

Comment:
should it be `redfish_host`? We already have `redfish_port`...

Reply:
Yes, I will rename it. :)

Reply:
@ricardoas: As `host = 0.0.0.0` works on Windows (as @ffdarkpenguin said), is this pull request still necessary? :thinking:

Reply:
Anyway, fixed!

Reply:
Not sure, @llvieira... any ideas, @danielreed? Launching the service for 0.0.0.0 on Windows worked for @ffdarkpenguin... do you see any other reason to leave this configurable?